app/views/site/_study_download_data.html.erb

```diff
@@ -187,8 +193,9 @@
       <table class="table table-striped tabled-condensed study-data-table" id="fastq-table">
         <thead>
         <tr>
-          <th class="col-sm-5">Filename</th>
-          <th class="col-sm-5">Description</th>
+          <th class="col-sm-4">Filename</th>
+          <th class="col-sm-4">Description</th>
+          <th class="col-sm-2">Species</th>
```

Comment:
I see "Species/Assembly" as the table header in other tables that have this column.  Should "Species" be updated to "Species/Assembly" here as well, for consistency?

Reply:
Yup - will do